sdk/src/main/java/com/uid2/sdk/data/IdentityPackage.kt

```diff
@@ -0,0 +1,12 @@
+package com.uid2.sdk.data
+
+/**
+ * The lifecycle of the Identity will cause the UID2Identity to change, along with potentially becoming invalid in some
+ * circumstances. This class represents combines the different possible states.
+ */
+data class IdentityPackage(
+    val valid: Boolean,
+    val errorMessage: String?,
+    val identity: UID2Identity?,
+    val status: IdentityStatus
```

Comment:
Since `errorMessage` and `identity` are `nullable`, should they be last in the arg list?

Reply:
I kept it as-is to match the Swift, although don't feel super strongly.